xml/http_servers.xml

```diff
@@ -1506,6 +1506,17 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:nginx:nginx:{service.version}"/>
   </fingerprint>
 
+  <fingerprint pattern="^TeamSpeak Server ([\d.]+)$">
+    <description>TeamSpeak HTTP server</description>
+    <example service.version="3.13.6">TeamSpeak Server 3.13.6</example>
+    <example service.version="3.13.3">TeamSpeak Server 3.13.3</example>
+    <param pos="0" name="service.vendor" value="TeamSpeak"/>
+    <param pos="0" name="service.product" value="TeamSpeak"/>
+    <param pos="0" name="service.family" value="TeamSpeak"/>
```

Comment:
Is TeamSpeak a `service.family` given they have no other products?

Reply:
Good point, will remove.